types/backbone-associations/index.d.ts

```diff
@@ -1,8 +1,8 @@
-// Type definitions for Backbone-associations 0.6.4
+// Type definitions for Backbone-associations 0.6
 // Project: http://dhruvaray.github.io/backbone-associations
 // Definitions by: Craig Brett <https://github.com/craigbrett17>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 2.8
```

Comment:
Why did this need to change? This package wasn't otherwise changed.

Reply:
Hi @DanielRosenwasser 

The `backbone-associations` library depends on `backbone` which in turn depends on `underscore` which we just modified and it now requires TypeScript 2.8 to operate.

That change forces TypeScript version change to `backbone-associations` (and many other libraries as you can see in the changed files)

If you look at @jgonggrijp comment below, you will see that this change won't affect current users as they can choose to import a version that supports the TypeScript version they actually use.
 
> @jgonggrijp You are correct. People who are using old versions of typescript should be using `@types/underscore@ts2.6` (to use TS 2.6 as an example) to acquire types, in which case they will not even see this new version.

If you believe that this is not an issue, please mark the requested changes as resolved, otherwise please let us know.

Thanks!

Reply:
@regevbr I agree with you, but that quote belongs to `@sandersn` (back-quoting in order to not needlessly attract his attention).

@DanielRosenwasser you can find the quote in question over here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/36230#issuecomment-502837876 which in turn was a reply to my question over here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/36230#pullrequestreview-250386425. Please observe that my question mentions *package versions*, which indeed should not be bumped in the packages that didn't change. The *TypeScript versions* did need to change because of the "viral" cascade that @regevbr described.

Reply:
@jgonggrijp my bad for the quote attribution :-)